packages/semantic-model/src/fix-api-json.ts

```diff
@@ -25,3 +28,16 @@ export function addViewDefaultAggregation(
     }
   }
 }
+
+export function fixSelfImplementingClass(
```

Comment:
This is becoming more and more like the compiler in dts-generator in UI5-TypeScript.
We will need to re-evaluate in the future how to merge these projects.